doc/developer-guides/hld/hv-memmgt.rst

```diff
@@ -62,7 +61,10 @@ Hypervisor Physical Memory Management
 In the ACRN, the HV initializes MMU page tables to manage all physical
 memory and then switches to the new MMU page tables. After MMU page
 tables are initialized at the platform initialization stage, no updates
-are made for MMU page tables.
+are made for MMU page tables except hv_access_memory_region_update is called.
+However, the memory region which hv_access_memory_region_update will update
+must have not been accessed by ACRN hypervisor because there's no flush
```

Comment:
Should we say?
hv_access_memory_region could only be accessed by BSP and must not be accessed after AP is started.

Reply:
No. Actually, hv_access_memory_region_update  would be called after AP is started. Even after SOS has booted. But there's only one case which was NPK for debug use. Other cases were all called on BSP before AP was started. Thanks.

Reply:
Does ACRN access NPK which call hv_access_memory_region_update? I suppose yes. Then
is "However, the memory region which hv_access_memory_region_update will update
must have not been accessed by ACRN hypervisor because there's no flush" conflict with this behavior?